api/services/wwwConfig.js

```diff
@@ -123,12 +123,10 @@ function _getAPISystemIntegration(bag, next) {
             'No systemIntegration found for query: ' + query)
         );
 
-      var defaultAPIIntegration = _.findWhere(
+      bag.publicApiIntegration = _.findWhere(
         systemIntegrations, {name: 'api'});
-      var serviceAPIIntegration = _.findWhere(
-        systemIntegrations, {name: bag.config.apiUrlIntegration});
-
-      bag.apiSystemIntegration = serviceAPIIntegration || defaultAPIIntegration;
```

Comment:
i guess u can keep this. `bag.internalApiIntegration = bag.internalApiIntegration || bag.publicApiIntegration;`

Reply:
ok, I'll update